lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -89,6 +89,11 @@ e2function number entity:id()
 	return this:EntIndex()
 end
 
+e2function number entity:creationID()
+	if not IsValid(this) then return 0 end
```

Comment:
No, however it matches how the rest of the file is arranged.

I could re-write most of the file in that format instead in another PR, I feel like that might be faster but I am unsure. I do not experience in benchmarking Lua.. or really at all.